rules/windows/lateral_movement_executable_tool_transfer_smb.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/11/10"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation or change of a Windows executable file over network shares. Adversaries may transfer tools or
+other files between systems in a compromised environment.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Lateral Executable Transfer Over SMB"
```

Rule name needs minor tweaking, below are a few suggestions. 

```suggestion
name = "Incoming File Transfer Over SMB"
```

```suggestion
name = "Tool Transfer Incoming Lateral Movement (SMB)"
```

---

agree, MITRE name it Lateral Tool Transfer, will change it as is (short and make sense).